frontend/js/redux/actions/result-actions.js

```diff
@@ -101,6 +109,8 @@ export function createElectionReminder(SurveyResultId, payload, callback) {
         const body = {};
         payload.email && (body.email = payload.email);
         payload.phone && (body.phone = payload.phone);
+        payload.name && (body.name = payload.name);
```

Comment:
This is to get around an oddity in Joi validation.  Undefined or a blank string is considered to be a valid string by `Joi.string()`.